enterprise/backend/src/metabase_enterprise/sandbox/models/group_table_access_policy.clj

```diff
@@ -122,6 +123,23 @@
 (log/trace "Installing additional EE pre-update checks for Card")
 (reset! card/pre-update-check-sandbox-constraints update-card-check-gtaps)
 
+(defenterprise upsert-sandboxes!
```

Comment:
Is it worth mentioning that this should be called in a transaction in the doc string?

Reply:
Calling it in a transaction is specific to the usage in the `/api/permissions/graph` handler. In theory this could be called elsewhere in the future if we need to create/update sandboxes and a transaction might not be necessary, so I think I'll leave this out.